### ISSUE TITLE
[todo]  Improve Style Corridors

### ISSUE THREAD

**Issue by TeamSpen210** [BEEmod/BEE2-items#530]
Most corridors haven't been updated to use compiler features and attribute variants, as well as being simple retextures.
- 50s Style
  - [ ] SP Entry 1
  - [ ] SP Entry 2
  - [ ] SP Entry 3
  - [ ] SP Entry 4
  - [ ] SP Entry 5
  - [ ] SP Entry 6
  - [ ] SP Entry 7
  - [ ] SP Exit 1 
  - [ ] SP Exit 2
  - [ ] SP Exit 3
  - [ ] SP Exit 4
  - [X] Coop Spawn
  - [ ] Coop Exit 1
  - [ ] Coop Exit 2
  - [ ] Coop Exit 3
  - [ ] Coop Exit 4
- 50s Style
  - [x] SP Entry 1
  - [ ] SP Entry 2
  - [ ] SP Entry 3
  - [ ] SP Entry 4
  - [ ] SP Entry 5
  - [ ] SP Entry 6
  - [ ] SP Entry 7
  - [ ] SP Exit 1 
  - [ ] SP Exit 2
  - [ ] SP Exit 3
  - [ ] SP Exit 4
  - [ ] Coop Spawn
  - [ ] Coop Exit 1
  - [ ] Coop Exit 2
  - [ ] Coop Exit 3
  - [ ] Coop Exit 4
- 70s Style
  - [ ] SP Entry 1
  - [ ] SP Entry 2
  - [x] SP Entry 3
  - [ ] SP Entry 4
  - [ ] SP Entry 5
  - [ ] SP Entry 6
  - [ ] SP Entry 7
  - [ ] SP Exit 1 
  - [ ] SP Exit 2
  - [ ] SP Exit 3
  - [ ] SP Exit 4
  - [ ] Coop Spawn
  - [ ] Coop Exit 1
  - [ ] Coop Exit 2
  - [ ] Coop Exit 3
  - [ ] Coop Exit 4
- 80s Style
  - [ ] SP Entry 1
  - [ ] SP Entry 2
  - [ ] SP Entry 3
  - [ ] SP Entry 4
  - [ ] SP Entry 5
  - [ ] SP Entry 6
  - [ ] SP Entry 7
  - [ ] SP Exit 1 
  - [ ] SP Exit 2
  - [ ] SP Exit 3
  - [ ] SP Exit 4
  - [ ] Coop Spawn
  - [ ] Coop Exit 1
  - [ ] Coop Exit 2
  - [ ] Coop Exit 3
  - [ ] Coop Exit 4
- P1 Style
  - [ ] SP Entry 1
  - [ ] SP Entry 2
  - [ ] SP Entry 3
  - [ ] SP Entry 4
  - [ ] SP Entry 5
  - [ ] SP Entry 6
  - [ ] SP Entry 7
  - [ ] SP Exit 1 
  - [ ] SP Exit 2
  - [ ] SP Exit 3
  - [ ] SP Exit 4
  - [X] Coop Spawn
  - [X] Coop Exit 1
  - [X] Coop Exit 2
  - [X] Coop Exit 3
  - [X] Coop Exit 4
- Overgrown Style
  - [X] SP Entry 1
  - [X] SP Entry 2
  - [X] SP Entry 3
  - [X] SP Entry 4
  - [X] SP Entry 5
  - [X] SP Entry 6
  - [X] SP Entry 7
  - [ ] SP Exit 1 
  - [ ] SP Exit 2
  - [ ] SP Exit 3
  - [ ] SP Exit 4
  - [X] Coop Spawn
  - [X] Coop Exit 1
  - [X] Coop Exit 2
  - [X] Coop Exit 3
  - [X] Coop Exit 4
- BTS Style
  - [x] SP Entry 1
  - [x] SP Entry 2
  - [x] SP Entry 3
  - [ ] SP Entry 4
  - [ ] SP Entry 5
  - [ ] SP Entry 6
  - [ ] SP Entry 7
  - [x] SP Exit 1 
  - [ ] SP Exit 2
  - [ ] SP Exit 3
  - [ ] SP Exit 4
  - [ ] Coop Spawn
  - [ ] Coop Dissasembly Room
  - [ ] Coop Exit 1
  - [ ] Coop Exit 2
  - [ ] Coop Exit 3
  - [ ] Coop Exit 4

**Comment by Konclan** [comment]
dang, that's a lot

**Comment by TheReviewers** [comment]
One Of The Portal 1 Entry Should Be An Elevator Straight To The Chamber...

**Comment by TeamSpen210** [comment]
That's doable.

**Comment by CrashingScience** [comment]
Yes, there are currently only 1 entry and exit BTS corridors, and no multiplayer ones. The BTS separated checkpoint has no style too.

**Comment by Luke18033** [comment]
I don't think this is fixed.

**Comment by Konclan** [comment]
From @LautaroL20
Bad squarebeams shadows in Co-op Exit corridor
![screenshot](https://cloud.githubusercontent.com/assets/12244376/13754583/54825a5c-e9f6-11e5-83dd-51d932b9dec6.jpg)

**Comment by Konclan** [comment]
From @Enterprise
Add better Lighting on the catwalk exit from some Old Apertures

This is how actually the exit catwalks looks in some Old Apertures from the single-player mode:

![20160313103449_1](https://cloud.githubusercontent.com/assets/15699236/14091937/96622fdc-f519-11e5-80c7-2cff6ca4e16e.jpg)

![20160313093729_1](https://cloud.githubusercontent.com/assets/15699236/14091957/b732ea8a-f519-11e5-8158-0e96e7ada3c4.jpg)

And this is the Light i suggest to add:

![20160313092706_1](https://cloud.githubusercontent.com/assets/15699236/14091969/d57b9230-f519-11e5-8f5a-db7f9041e937.jpg)

**Comment by Konclan** [comment]
From @ghost:

Old Aperture catwalks need variation. They are all straight - left - straight

They should all be different, every entry/exit corridor should have its own catwalk :3

**Comment by Konclan** [comment]
https://github.com/BEEmod/BEE2.4/blob/master/FAQ.md#why-is-bts-being-removed
